components/components/view-modes/highlighted-small/node--highlighted-small.html.twig

```diff
@@ -1,30 +1,33 @@
-<?php
+{#
 /**
  * @file
  * This file creates the 'highlighted-small' view mode for our styleguide.
  */
-?>
-
-<style>
-  <?php include $_SERVER['DOCUMENT_ROOT'] . '/base.css'; ?>
-  <?php include $_SERVER['DOCUMENT_ROOT'] . '/components/view-modes/highlighted-small/style.css'; ?>
-</style>
-
-<script>
-  <?php include 'script.js'; ?>
-</script>
+#}
 
 <article role="article" class="node node--type-recipe node--view-mode-highlighted-small" about="/pages/recipe">
   <div class="node__content">
+
     <div class="field field--name-field-image field--type-image field--label-hidden field__item">
       <a href="/pages/recipe" rel="bookmark">
-        <?php include $_SERVER['DOCUMENT_ROOT'] . '/components/image-styles/highlighted-small.php'; ?>
+        {% include '../../image-styles/highlighted-small.html.twig' %}
       </a>
     </div>
-    <h2>
-      <a href="/pages/recipe" rel="bookmark">
-        <span class="field field--name-title field--type-string field--label-hidden"><?php print $title; ?></span>
-      </a>
-    </h2>
+
+    <div class="node__meta">
+      <div class="field field--name-field-difficulty field--type-list-string field--label-inline">
+        <div class="field__label">Difficulty</div>
+          <div class="field__item">Hard</div>
+        </div>
+      <h2>
```

Comment:
I'm not sure where this markup comes from, but could we add a class to avoid using the `h2` directly in the CSS?

Reply:
yes, .node__title is now added, it should have been there already.

The markup is coming from what is being output by Drupal here: http://dev-ootb.pantheonsite.io

I'm trying to replicate the same markup, so we can have a good guess at what we are going to get.